.github/issue-rules.yml

```diff
@@ -344,6 +344,12 @@ rules:
 - valueFor: 'Component'
   contains: 'sdk-typescript'
   addLabels: ['sdk-typescript']
+- valueFor: 'Component'
+  contains: 'Test is flaky'
+  addLabels: ['flaky']
+- valueFor: 'Component'
+  contains: 'Test is continually failing'
+  addLabels: ['failing']
```

Comment:
I'd cut this label or make it `permared` (or something similar). We're already applying the `test-failures` label, so this doesn't add much